.github/workflows/create-dns-record.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# resource group of DNS Zone for the hack
+DNS_ZONE_RG=TLD
+
+# name of DNS Zone for the hack
+DNS_ZONE_NAME=aks-sb.com
+
+if [ -z "$ASB_TEAM_NAME" ]
+then
+  echo "ASB_TEAM_NAME name is required"
+  exit
+fi
+
+if [ -z "$ASB_AKS_PIP" ]
+then
+  echo "ASB_AKS_PIP is required"
+  exit
+fi
+
```

Comment:
will this update an existing DNS name?  If not, should we check and delete?

Reply:
If the name is the same and ip address is the same, the command will fail.
If the name is the same, but the ip address is different, it'll add another ip address to the name.
I'll add notes to this task, https://github.com/retaildevcrews/asb-hack/issues/324, to do the check and delete behavior.